lib/sms/sms.c

```diff
@@ -199,6 +214,14 @@ static void sms_at_cmd_handler_cds(const char *at_notif)
  */
 static void sms_at_cmd_handler_cms(const char *at_notif)
 {
+#if CONFIG_SMS_CMS_STRERROR
+	int err = 0;
+	int ret = sscanf(at_notif, "+CNEC_ESM: %d", &err);
```

Comment:
What does it mean to parse `+CNEC_ESM` in `+CMS` handler? Have you verified this?